ssh/tailssh/incubator.go

```diff
@@ -699,9 +701,40 @@ func fileExists(path string) bool {
 	return err == nil
 }
 
+// loginArgs returns the arguments to use to exec the login binary.
+// It returns nil if the login binary should not be used.
```

Comment:
the caller on line 222 seems to care about non-empty length, not nil vs non-nil

make consistent one way or another? (docs & caller)